src/main/java/com/tokyoprogrammer/aqitracer/controller/AqitracerController.java

```diff
@@ -35,13 +47,59 @@ public HttpResponse healthCheck() {
 
     @Get
     @Path("/api/v1/aqi")
-    public String getAqi(@Param("latlng") String latlng, @Param("provider") AQI_PROVIDER provider) {
-        return ImmutableMap.of("coordinate", latlng, "provider", provider).toString();
+    public HttpResponse getAqi(@Param("current") String current, @Param("bound") String bound,
+                               @Param("provider") AQI_PROVIDER provider) {
```

Comment:
Not sure whether the provider parameter is needed or not. On client side, there is no way to provide the provider parameter properly. 
(e.g. in case of a user use this app in the US, will the client need to choose proper provider and send it to API? Then how can the client determine whether the decision is correct or not? In case of the user in the US search different area(such as china)? Then how the client know which kind of providers are existing under certain condition and managing AQI data?)
In my opinion, providers should be determined in server side and provide proper AQI data based on latlng info as only server will know which kind of data sources are managed and which kind of data are managed under which condition. Please discuss about this.